pkg/apis/core/types.go

```diff
@@ -157,6 +157,31 @@ type VolumeSource struct {
 	// CSI (Container Storage Interface) represents ephemeral storage that is handled by certain external CSI drivers (Beta feature).
 	// +optional
 	CSI *CSIVolumeSource
+	// Ephemeral represents a volume that is handled by a normal storage driver (Alpha feature).
```

Comment:
`cluster storage driver` maybe?  `normal` doesn't sound right.

Reply:
Ack. That's better.